pom.client.xml

```diff
@@ -630,6 +649,7 @@
       <modules>
         <module>./eng/spotbugs-aggregate-report</module>
         <module>./eng/code-quality-reports</module>
+        <module>./eng/testcoverage</module>
```

Comment:
For naming consistency, can this be 'jacoco-test-coverage'

Reply:
Done